command/v6/services_command_test.go

```diff
@@ -171,7 +171,7 @@ var _ = Describe("services Command", func() {
 					)
 				})
 
-				It("displays all the services in the org & space & warnings", func() {
+				It("displays all the services and apps in alphanumeric sorted order together with the org & space & warnings", func() {
```

Comment:
There should be some kind of test ensuring that the sorting described here is actually occurring.

Reply:
Agree. We've moved around some service instances, and apps.